src/components/PostSection/index.jsx

```diff
@@ -99,7 +109,16 @@ const PostSection = ({
       textColor: '#e03131'
     }
   ];
-
+  const optionReport = [
+    {
+      svg: reportIcon,
+      text: 'Report Post',
+      fun: () => {
+        reportPost();
+      },
+      textColor: '#000'
+    }
+  ];
```

Comment:
why you add this variable and use it?
although you already added it to the options array